src/3rdparty/walkontable/src/renderer/rowHeaders.js

```diff
@@ -81,7 +81,6 @@ export default class RowHeadersRenderer extends BaseRenderer {
         const TH = orderView.getCurrentNode();
 
         TH.className = '';
-        TH.removeAttribute('style');
```

Comment:
Are you sure? We're clearing the TH styles here. What if someone added `style=""` with `afterGetRowHeaders`?

Reply:
Good point 🤔

Reply:
@wojciechczerniak It's hard to find alternative solution for me. Repositioning first element sounds like hack. However, maybe changing style by `afterGetRowHeaders` isn't so problematic? It seems that callback is called on every render and pure function may give absolutely satisfactory results.

Please look at below examples. Config: 

```js
    data: Handsontable.helper.createSpreadsheetData(4, 4),
    contextMenu: true,
    rowHeaders: true,
    colHeaders: true,
    trimRows: true,
    afterGetRowHeader(row, th) {
      if (row % 2 === 0) {
        th.style.background = 'lightblue';

      } else {
        th.style.background = 'lightcyan';
      }
    },
``` 

1. Adding row by context-menu.

https://user-images.githubusercontent.com/141330/108200409-bab03600-711e-11eb-9bb3-8da91b6b2194.mp4

2. Trimming row.

https://user-images.githubusercontent.com/141330/108200746-2e524300-711f-11eb-9734-0ef82870f6df.mp4

Reply:
@wszymanski  I know it's not really a real-life example, but I'm afraid this could potentially be a breaking change:

https://codesandbox.io/s/vanilla-handsontable-pr-forked-jxv1j

Select a cell, and traverse through some of them using <kbd>ENTER</kbd> - the styles don't get cleared, as they do currently.

Reply:
@jansiegel There is a problem with your code, because user shouldn't overwrite `style` attribute. It removes also our styles, which are applied by `AutoRowSize` plugin. It shouldn't be supported. User probably should use classes?

However, I understand the idea. Maybe all styles, beside used `height` declaration, should be removed 🤔? Please look at https://github.com/handsontable/handsontable/pull/7529/commits/3cb1a5b7438083937db71c6ceb1effd2f6775cf6. I'm not convinced to this idea, but it may be worth worth considering 🤔? WDYT? @wojciechczerniak @jansiegel

Reply:
I _guess_ we could do something like that, and it would not _technically_ be a breaking change I think, but it might potentially break some user implementations.

@wojciechczerniak ?

Reply:
Since this issue has been lingering here since April, I'll pass the question above to @warpech ;)

Reply:
I am against solutions that read from DOM. We have one way of setting row heights explained on the page https://handsontable.com/docs/row-height/. The AutoRowSize plugin adheres to it by using the `modifyRowHeight` hook. Walkontable has this information and it must be able to write it to DOM correctly, without reading it from DOM.

I might be naively overlooking something, but it seems to me that it is not this line or file that is wrong: https://github.com/handsontable/handsontable/blob/72120c2be278140a094338125e0d1a9bbc49f78f/src/3rdparty/walkontable/src/renderer/cells.js#L92

Rather, this line seems to be not applied correctly: